publications/publication-guidelines.md

```diff
@@ -0,0 +1,9 @@
+# CHAOSS Data Science Publication Guidelines
+
+One goal of the CHAOSS Data Science Working Group (WG) is to begin publishing the research and outcomes from our projects so that people can learn from our findings while also raising the visibility of CHAOSS and this WG. Our current list of projects can be found as [Issues labelled with "Project"](https://github.com/chaoss/wg-data-science/issues?q=is%3Aissue%20state%3Aopen%20label%3Aproject) in the Data Science WG repository, and we have an issue template that anyone can use to propose a new project.
+
+Open source program offices, community managers, and other people working in corporate open source environments have been the biggest audiences for CHAOSS tools and metrics, so to make it easier for these people to consume our research, we are focusing on producing reports that are in the style of white papers, like what you can find published by [Linux Foundation Research](https://www.linuxfoundation.org/research). Ideally, we would like work with LF Research to release our reports, since we are an LF project. However, not everything we publish be appropriate for an LF Research report, so alternatively, some will be published as reports on the CHAOSS site. Good reports to model ours after are the LF / Harvard [Census II](https://www.linuxfoundation.org/hubfs/Research%20Reports/lfr_harvard_censusII_mar2022_042824b.pdf?hsLang=en) and [Census III](https://www.linuxfoundation.org/hubfs/LF%20Research/lfr_censusiii_120424a.pdf?hsLang=en) reports, since they are well-researched, but presented in a style that is accessible for a wide audience (note that we expect our reports to be much shorter). 
```

Comment:
Change: However, not everything we publish be appropriate for an LF Research report
To However, not everything we publish will be appropriate for an LF Research report,